SPECS/cri-o/cri-o.spec

```
@@ -102,6 +111,15 @@ This package provides the CRI-O container runtime configuration for kubeadm
 %setup -q
 tar -xf %{SOURCE1} --no-same-owner
 %patch0 -p1 -b .files
+%patch1 -p1
+%patch2 -p1
+%patch3 -p1
+%patch4 -p1
+%patch5 -p1
+%patch6 -p1
+%patch7 -p1
+%patch8 -p1
+%patch9 -p1
```

NIT: we might as well use `autopatch -p1`
```suggestion
%autopatch -p1
```